requirements.txt

```diff
@@ -45,7 +45,7 @@ mergedeep~=1.3
 # TODO: remove this when v3io-frames no longer requires it
 flake8~=3.8
 # 3.2 iguazio system uses 0.9.x - limiting to only patch changes
-v3io-frames~=0.9.5
+v3io-frames~=0.9.6
```

Comment:
So now that frames does not require flake8 anymore, we can remove flake8 from here, please remove it (and the comment above it)